01-Fundamentals-part-2/assignments2.js

```diff
@@ -109,4 +109,34 @@ if (mark.calcBmi() < john.calcBmi()) {
       john.fullName
     }'s (${john.bmi.toFixed(2)}))`
   );
+}*/
+console.log("Fundamentals part 2. Coding challenge #4");
+
+const bills = [22, 295, 176, 440, 37, 105, 10, 1100, 86, 52];
+const tips = [];
+const totals = [];
+let calcTip = (billValue) => {
+  if (billValue > 50 && billValue < 300) {
+    return billValue * 0.15;
+  } else return billValue * 0.2;
+};
+for (let i = 0; i < bills.length; i++) {
+  tips.push(calcTip(bills[i]));
+  totals.push(bills[i] + tips[i]);
 }
+console.log("Bills are", bills);
+console.log("Tips are", tips);
+console.log("Totals are", totals);
+
+//BONUS
+const calcAverage = function (arr) {
```

Comment:
Looks good! 
BONUS task 1: please create `calcAverage2` that would use arrow functions (instead of traditional function expression), for loop (instead of while) and would return value (instead of console.log)

Reply:
Ok. Check it.